packages/docs/src/components/demo/CodeInput.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="code-input">
+    <input class="code-input__input" v-model="valueComputed" :style="{ width: `${textDivWidth}px` }" />
+    <div class="code-input__text" ref="textDiv">{{ valueComputed }}</div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { nextTick } from 'process'
+import { computed, ref, defineProps, defineEmits, watch, onMounted } from 'vue'
+
+const props = defineProps({
+  modelValue: { type: String },
+})
+
+const emit = defineEmits(['update:modelValue'])
+
+const valueComputed = computed({
+  get () { return props.modelValue },
+  set (v) { emit('update:modelValue', v) },
+})
+
+const textDiv = ref<HTMLDivElement>()
+const textDivWidth = ref(0)
+const inputPaddingX = 4
+const inputPaddingXPx = `${inputPaddingX}px`
```

Comment:
```suggestion
```
I think you've been practicing with `v-bind` in scss))

Reply:
I thought I will need these values in calculation.